molecule/alternative/playbook.yml

```diff
@@ -5,6 +5,8 @@
   roles:
     - ansible-prometheus
   vars:
+    # This version should match prepare.yml
+    prometheus_version: 2.14.0
```

Comment:
The idea was that alternative tests are downloading prometheus binary in `prepare.yml` playbook and then use a binary as a local one. For such mode of operation `prometheus_version` shouldn't matter as it is set to download the correct tarball in `download prometheus binary to local folder` task.

However, it seems we need to adjust this mode of operation as we are also using `prometheus_version` in systemd service template.

Reply:
Ahh, ok. Now I get it.

I guess a solution would be to keep the alternative test version up to at least the oldest supported version.

Reply:
yes, that sounds good :+1: